app/lib/methods/sendFileMessage.ts

```diff
@@ -130,9 +142,9 @@ export function sendFileMessage(rid, fileInfo, tmid, server, user) {
 				}
 			});
 
-			uploadQueue[fileInfo.path].catch(async error => {
+			uploadQueue[fileInfo.path].catch(async (error: unknown) => {
```

Comment:
Just a tip, in these cases as the error is not instantiated, it is not necessary to return its type.